.github/workflows/community.yml

```diff
@@ -0,0 +1,29 @@
+name: Community tests
+on:
+  pull_request:
+env:
+  PUB_ENVIRONMENT: bot.github
+
+jobs:
+  drift:
+    # Tests maintained by oss@simonbinder.eu
+    name: "Tests for pkg:drift"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: dart-lang/setup-dart@v1.3
+      - uses: actions/checkout@v2
+      - run: dart pub get
+        working-directory: tool
+
+      - uses: actions/checkout@v2
+        with:
+          repository: simolus3/moor
+          ref: latest-release
+          path: client_tests
```

Comment:
Should this be `client_tests/moor`? Or just call it `moor_tests`?

Reply:
It shouldn't make much of a difference since we'll probably want to put different community tests into different jobs, so they'll all run from a clean state? But either way I've just put the moor checkout into a project-specific folder.